tasty/graphs.py

```diff
@@ -144,3 +146,81 @@ def get_namespaced_term(ontology: Graph, term: str) -> Union[URIRef, bool]:
         ns = potential_namespaces[0]
         return ns[term]
     return False
+
+
+def print_as_hayson(graph: Graph):
+    """
+    Return the Hayson encoding of an rdf graph
+    :param graph: [rdflib.Graph]
+    :return:
+    """
+    hayson = {
+
+        "meta": {"ver": "3.0"},
+        "cols": [],
+        "rows": []
+    }
+    json_ld = graph.serialize(format='json-ld').decode('utf-8')
+    data = json.loads(json_ld)
```

Comment:
If you can, I would just do the following since the `json_ld` variable is only used once.
```
json_ld_str = json.loads(graph.serialize(format='json-ld').decode('utf-8'))
```

Reply:
👍